cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -94,6 +94,7 @@ INVALID_TICKET=Ticket ''{0}'' not recognized
 INVALID_SERVICE=Ticket ''{0}'' does not match supplied service. The original service was ''{1}'' and the supplied service was ''{2}''.
 INVALID_PROXY_CALLBACK=The supplied proxy callback url ''{0}'' could not be authenticated.
 UNAUTHORIZED_SERVICE_PROXY=The supplied service ''{0}'' is not authorized to use CAS proxy authentication.
+UNSATISFIED_AUTHN_POLICY=Service access denied due to an unsatisfied authentication policy.
```

Comment:
`UNKNOWN` will need to stay because it is used for any unrecognized `handlerErrors` within an `AuthenticationException`